sass/themes/shop/2019/shop19.scss

```diff
@@ -0,0 +1,63 @@
+$item-grey: #ddd;
+// $colour-red: #e81962;
+// ==========================================================================
```

Comment:
these colours are also specified in `_shop19-components.scss`
I suppose if _shop19-components.scss is imported at the end of this file you can use the variables from here and also any varibales and mixins available through the base theme since they're imported here. (saw you didn't use the breakpoint mixin in _shop19-components.scss )

Reply:
Removing the colour-red redefinition to use CR19's version, and using a CR19 light-grey to set the value of $item-grey. 

The shop theme doesn't actually use this variable at all, I guess we just needed to define whatever was in the base theme for the Shop18 theme? Tempted to just remove it (as MRM aren't using this theme as SASS), but will keep it for now.

Reply:
yeah I think item-grey is used for our forms and comes from the base theme. 
I see it was added to the shop18 theme, but not sure where it's used. Suppose we'll keep it as it won't do any harm anyway.